Movie.h

```diff
@@ -0,0 +1,82 @@
+#pragma once
+#include <iostream>
+#include <string>
+using namespace std;
+
+class Movie
+{
+public:
+	int id;
+	string title;
+	string ranking;
+	int release_date;
+	int tickets;
+	int ticket_price;
+	string comment;
+
+	int getId()
+	{
+		return id;
+	}
+	string getTitle()
+	{
+		return title;
+	}
+	string getRanking()
+	{
+		return ranking;
+	}
+	int getReleaseDate()
+	{
+		return release_date;
+	}
+	int getNumberOfTickets()
+	{
+		return tickets;
+	}
+	int getTicketPrice()
+	{
+		return ticket_price;
+	}
+	string getComment()
+	{
+		return comment;
+	};
+
+	Movie() {};
+
+	enum filmType
+	{
+		Action, Comedy, Drama, Fantasy
+	};
+
+	void calculateProfit()
+	{
+		int a = tickets;
+		int b = ticket_price;
+		int c = a * b;
+		cout << "Profit for a day : " << c << endl;
+	};
+
+	Movie(int aId, string aTitle, string aRanking, int aRelease_date, int aTickets, int aTicket_price, string aComment)
```

Comment:
Change names due to naming conventios 

https://google.github.io/styleguide/cppguide.html#Naming